pyuvdata/uvflag/uvflag.py

```diff
@@ -1856,7 +1856,10 @@ def _select_preprocess(
             blt_inds = uvutils._get_iterable(blt_inds)
             if np.array(blt_inds).ndim > 1:
                 blt_inds = np.array(blt_inds).flatten()
-            history_update_string += "baseline-times"
+            if self.type == "baseline":
+                history_update_string += "baseline-times"
+            else:
+                history_update_string += "times"
```

Comment:
I wonder if we should have a separate keyword `t_inds`, and error if you set `blt_inds` when not type `baseline`? Or maybe at least warn?

Reply:
Yeah, maybe. But at the same time time it only appears in functions that presumably a regular user would never call and the `blt_ind`/`t_ind` would just introduce another input/output to the function which end up slicing into the same arrays anyway. I can add it if you'd like though.

Reply:
This is a private helper method. If there's a concern about user input it should be handled in the public method that calls this one.